assets/css/style.css

```diff
@@ -451,4 +451,8 @@ form.checkout.woocommerce-checkout.dhp-exp {}
 /*Fixes*/
 div#dhp-embed {
     width: 100%;
-}  
+}
+
+.errorMessage {
```

Comment:
do we want to prefix Dintero class names with `dintero-` or some such things to avoid css naming collisions with other styles present in the merchant site?

Reply:
also aren't style names usually `written-like-this`?

Reply:
Fixed